pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -47,7 +47,13 @@ const (
 	typeService           = "Service"
 )
 
+type KubeDataInterface interface {
```

Comment:
nitpick: this interface can be private.

Reply:
done (I just had to move the mock into kubernetes package :) )